src/client.js

```diff
@@ -73,8 +78,11 @@ function Client(sockets, name, config) {
 		});
 	}
 }
+util.inherits(Client, EventEmitter);
```

Comment:
[This](https://nodejs.org/api/events.html#events_inheriting_from_eventemitter) shows that you'd need to call `EventEmitter.call(this);` as part of your `function Client(...)` function but I'm not sure why.